src/header/common/range.rs

```diff
@@ -0,0 +1,199 @@
+use std::fmt::{self, Display};
+use std::str::FromStr;
+
+use header::{Header, HeaderFormat, RangeUnit};
+use header::parsing;
+
+/// The `Range` header defined by HTTP/1.0.
```

Comment:
Really HTTP/1.0? [RFC 1945](https://tools.ietf.org/html/rfc1945) does not mention it.